powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/SignatureController.java

```diff
@@ -96,9 +98,11 @@ public ObjectResponse<CreatePersonalizedOfflineSignaturePayloadResponse> createP
      */
     @PostMapping("/offline/non-personalized/create")
     public ObjectResponse<CreateNonPersonalizedOfflineSignaturePayloadResponse> createNonPersonalizedOfflineSignaturePayload(@RequestBody ObjectRequest<CreateNonPersonalizedOfflineSignaturePayloadRequest> request) throws Exception {
-        logger.info("CreateNonPersonalizedOfflineSignaturePayloadRequest received: {}", request);
+        logger.info("CreateNonPersonalizedOfflineSignaturePayloadRequest received.");
+        logger.debug("CreateNonPersonalizedOfflineSignaturePayloadRequest received: {}", request);
         final ObjectResponse<CreateNonPersonalizedOfflineSignaturePayloadResponse> response = new ObjectResponse<>(offlineSignatureService.createNonPersonalizedOfflineSignaturePayload(request.getRequestObject()));
-        logger.info("CreateNonPersonalizedOfflineSignaturePayloadRequest succeeded: {}", response);
+        logger.info("CreateNonPersonalizedOfflineSignaturePayloadRequest succeeded.");
```

Comment:
here add action: createNonPersonalizedOfflineSignaturePayload and state: succeeded.